DOL.WHD.Section14c.Web/src/modules/components/sectionAppInfo/sectionAppInfoTemplate.html

```diff
@@ -60,7 +60,7 @@ <h2 class="form-section-header">Application Type</h2>
         <fieldset class="usa-fieldset-inputs form-question-answer">
             <ul class="usa-unstyled-list">
                 <li ng-repeat="response in responses.EstablishmentType">
-                    <input id="establishmentType_{{ response.id }}" type="checkbox" name="establishmentType" ng-value="response.id" ng-model="formData.establishmentType">
+                    <input id="establishmentType_{{ response.id }}" type="checkbox" name="stablishmentType_{{ response.id }}" ng-value="response.id" ng-checked="formData.establishmentType.indexOf(response.id) > -1" ng-click="toggleEstablishmentType(response.id)">
```

Comment:
Tiny typo here.  It doesn't look like the name is used anywhere so it shouldn't be a functional problem, but I'm calling it out because `stablishment` is funny to me.